composer.lock

```diff
@@ -4891,46 +4891,42 @@
         },
         {
             "name": "symfony/console",
-            "version": "v5.4.1",
+            "version": "v6.0.2",
```

Comment:
This here is triggering the BC warning. Im not quite sure but i would say this is no bc for us. If we would say it's a BC then we could not support 3 Major Versions. If we would go the route to only support 1 major version, it would be fine from my site and then this would be a real bc break.